rop.spec

```diff
@@ -0,0 +1,56 @@
+Name:      rop
+Version:   0.0.1
+Release:   0%{?dist}
+Summary:   Install Foreman using containers
+
+License:   GPL-2-only
+URL:       https://github.com/theforeman/foreman-quadlet
+Source:    https://github.com/theforeman/foreman-quadlet/releases/download/%{version}/%{name}-%{version}.tar.gz
+
+BuildArch: noarch
+Requires:  ansible-collection-ansible-posix
+Requires:  ansible-collection-community-crypto
+Requires:  ansible-collection-community-general
+Requires:  ansible-collection-community-postgresql
+Requires:  ansible-collection-containers-podman >= 1.14.0
```

Comment:
sidequest: neither of these is in EL-proper (they are in EPEL, but that doesn't really help us long term).

- should we vendor them at build time (like we do with puppet modules in foreman-installer)?
- I'd prefer not to package them "properly", as that would make them available *outside* rop, and people might think they can use them for other things and claim support

Reply:
I had similar concerns and I'm not sure yet. They just happened to be in EPEL and allowed me to ignore the problem at first.

Reply:
Absolutely, it's not a problem that needs solving right now.

Reply:
Moved to https://github.com/theforeman/foreman-quadlet/issues/157